### ISSUE TITLE
Logstics Pipes incompatible with Simply Jetpacks

### ISSUE THREAD

**Issue by atm397** [RS485/LogisticsPipes#730]
Logistics Pipes requires Buildcraft version 6.3.5 which Simply Jetpacks is incompatible with. Simply Jetpacks will work if it's the latest version of Buildcraft but then Logistics Pipes won't work. Is there any reason for why it does this? Possibly an id conflict?

**Comment by bziemons** [comment]
There are already newer Logistics Pipes versions for BC 6.4 and BC 7. I don't see why it shouldn't be working.

**Comment by hron84** [comment]
@atm397 could you please describe which LogisticsPipes version do you use (or which modpack)?

I agree with @theZorro266, LP has a BC 6.4 and BC 7 version too, but if you use a modpack, please be careful with updating, since BC 6.4 got some updates and BC 7 can bring serious compatibility issues for too old mod versions - make sure you update all mods to a BC 6.4/BC7 compatible version (depending on which version do you choose).

**Comment by atm397** [comment]
I originally tried using 0.9.1.133 for logistic pipes with buildcraft 7.0.13 and bdlib 1.9.0.94. It crashed but worked fine when i pulled logistics pipes out. So then I clicked on other releases and tried 0.8.3.106 for buildcraft 6.3. I forgot how but somehow along the way it told me I had to upgrade to 6.3.2 so I did that and logistic pipes started working but not with simply jetpacks. The simply jetpacks version is 1.5.1.
